djtriggers/management/commands/clean_triggers.py

```diff
@@ -42,7 +42,9 @@ def handle_noargs(self, **options):
         # Delete each trigger
         for trigger in to_be_deleted:
             # Delete the specific trigger information
-            table = settings.DJTRIGGERS_TYPE_TO_TABLE[trigger.trigger_type]
+            table = getattr(settings.DJTRIGGERS_TYPE_TO_TABLE, trigger.trigger_type, None)
+            if table is None:
+                continue
```

Comment:
It's not really wanted, but we can't delete them if we don't know the type, because there is a foreign key to the main triggers table from the specific trigger table. Deleting the trigger without deleting that one first will just throw an error anyway.